tests/integration/022_templates/main.tf

```diff
@@ -0,0 +1,44 @@
+data "env0_template" "github_template" {
+  name = "Github Integrated Template"
+}
+
+resource "env0_template" "github_template1" {
+  name                                    = "Github Test-111"
```

Comment:
Please make sure to set the name randomly (like we do in order tests) - relevant for below resources as well.
Otherwise the integration tests would fail on consecutive runs

Reply:
good point.... /:
I guess I'll remove the output then. Can't it have it both ways I guess....